shotover-proxy/tests/cassandra_int_tests/prepared_statements.rs

```diff
@@ -119,4 +119,10 @@ where
     select_cross_connection(session, connection_creator).await;
     delete(session).await;
     use_statement(session).await;
+
+    if session.is(&[CassandraDriver::Scylla, CassandraDriver::CdrsTokio]) {
```

Comment:
The tests pass just fine on my machine when I remove this check.
Did something else incidentally fix the issue we were having with the datastax driver?